### PR TITLE
Bug 2057558: increase polling time of status report to reduce log spam

### PR DESCRIPTION
**Pull request by jbuchananr**
Description of the change:

This PR updates the log level to reduce log spam. Also, adds log-level knob on binary to allow users to adjust verbose level of logger.

Motivation for the change:

Reduce log spamming with poll status updates. 

Reviewer Checklist

 Implementation matches the proposed design, or proposal is updated to match implementation
 * Sufficient unit test coverage
 * Sufficient end-to-end test coverage
 * Docs updated or added to /docs
 * Commit messages sensible and descriptive

Signed-off-by: Jonathan Buchanan <jbuchananr19@gmail.com>

### CODE REVIEW

**Review by timflannagan — APPROVED** [review]

File: cmd/manager/main.go [hunk 1]
```diff
@@ -76,16 +77,26 @@ func main() {
...
 		version                 bool
+		loglvl                  string
 	)
 	flag.StringVar(&clusterOperatorName, "clusterOperatorName", "", "configures the name of the OpenShift ClusterOperator that should reflect this operator's status, or the empty string to disable ClusterOperator updates")
 	flag.StringVar(&defaults.Dir, "defaultsDir", "", "configures the directory where the default CatalogSources are stored")
 	flag.BoolVar(&version, "version", false, "displays marketplace source commit info.")
 	flag.StringVar(&tlsKeyPath, "tls-key", "", "Path to use for private key (requires tls-cert)")
 	flag.StringVar(&tlsCertPath, "tls-cert", "", "Path to use for certificate (requires tls-key)")
 	flag.StringVar(&leaderElectionNamespace, "leader-namespace", "openshift-marketplace", "configures the namespace that will contain the leader election lock")
+	flag.StringVar(&loglvl, "verbosity", "info", "Sets verbosity level of logger with default verbosity info level. See https://github.com/sirupsen/logrus for other verbosity levels.")
```
Comment (tylerslaton) [inline cmd/manager/main.go:88]: Could we change the argument to be `level` as `verbosity` is a different concept when it comes to logrus?

File: cmd/manager/main.go [hunk 2]
```diff
@@ -76,16 +77,26 @@ func main() {
...
 	flag.Parse()
-
 	logger := logrus.New()
+	// Set verbosity level
+	loglvl = strings.ToLower(loglvl)
+
```
Comment (tylerslaton) [inline cmd/manager/main.go:93]: This line is unneeded as the [ParseLevel function](https://github.com/sirupsen/logrus/blob/bdc0db8ead3853c56b7cd1ac2ba4e11b47d7da6b/logrus.go#L26) already does this. 
```suggestion

```

**Review by tylerslaton — COMMENTED** [review]
Looks good! Just a few comments.

File: cmd/manager/main.go (hunk 1 above)
Comment (jbuchananr) [reply cmd/manager/main.go:88]: Yep I like that.

File: pkg/status/status.go [hunk 3]
```diff
@@ -23,7 +23,7 @@ import (
 
 const (
 	// coStatusReportInterval is the interval at which the ClusterOperator status is updated
-	coStatusReportInterval = 20 * time.Second
+	coStatusReportInterval = 60 * time.Second
```
[the remark this answers is not in the document]
Comment (jbuchananr) [reply pkg/status/status.go]: Changed log level.

**Pull request merged**